tools/internal-rules/no-invalid-meta.js

```diff
@@ -61,6 +61,20 @@ function hasMetaDocsDescription(metaPropertyNode) {
     return metaDocs && getPropertyFromObject("description", metaDocs.value);
 }
 
+
+/**
+ * Checks whether the value of the property is empty or not
+ * @param {ASTNode} metaPropertyNode The `meta` ObjectExpression for this rule.
+ * @param {string}  type the property type to check
+ * @returns {boolean} `true` if propertie valye is empty else `false`
+ */
+function isEmptMetaStringProp(metaPropertyNode, type) {
```

Comment:
Please rename to isEmptyMetaStringProperty. 

(Spelling correction, no need to abbreviate property as we spell it out elsewhere.)

Reply:
sure

Reply:
Looks like this hasn't been changed yet?

Reply:
did you mean this one https://github.com/eslint/eslint/pull/13179/commits/ba92c63514ced0a517335495784fefac745e7500 ?

Reply:
"Empty" is still spelled incorrectly (missing the "y')

Reply:
done